docs/jwst/extract_1d/reference_files.rst

```diff
@@ -189,44 +209,56 @@ value.  The polynomial will then be evaluated at each pixel within the
 source extraction region for that column (row), and the fitted values will
 be subtracted (pixel by pixel) from the source count rate.
 
-.. include:: ../includes/standard_keywords.rst
-
-Example EXTRACT1D Reference
----------------------------
-The following JSON was taken as an example from reference
-jwst_nirspec_extract1d_0006.json::
+Example EXTRACT1D Reference File
+--------------------------------
+The following JSON was taken as an example from reference file
+jwst_niriss_extract1d_0003.json::
   {
-      "reftype": "EXTRACT1D",
-      "instrument": "NIRSPEC",
-      "telescope": "JWST",
-      "exp_type": "NRS_BRIGHTOBJ",
-      "pedigree": "GROUND",
-      "descrip": "NIRSpec NRS_BRIGHTOBJ extraction params for ground testing",
-      "author": "H.Bushouse",
-      "history": "Second draft 2016-Dec-07",
-      "useafter": "2001-01-01T00:00:00",
+      "REFTYPE": "EXTRACT1D",
+      "INSTRUME": "NIRISS",
+      "TELESCOP": "JWST",
+      "DETECTOR": "NIS",
+      "EXP_TYPE": "NIS_SOSS",
+      "PEDIGREE": "GROUND",
+      "DESCRIP": "NIRISS SOSS extraction params for ground testing",
+      "AUTHOR": "M.Wolfe, H.Bushouse",
+      "HISTORY": "This reference file is for used in Build 7.1 of the JWST Calibraton pipeline. The regions are rectagular and do not follow the trace.",
+      "USEAFTER": "2015-11-01T00:00:00",
       "apertures": [
         {
-        "id": "S1600A1",
+        "id": "FULL",
+        "region_type": "target",
+        "bkg_coeff": [[2014.5],[2043.5]],
+        "xstart":    4,
+        "xstop":  2044,
+        "ystart": 1792,
+        "ystop":  1972,
+        "dispaxis": 1,
+        "extract_width": 201
```

Comment:
This is the value that's in the file, but I'm not sure that this is what was intended.  The extraction width as specified by ystart and ystop is 181 pixels.  Specifying extract_width = 201 overrides that range (but uses the Y center location implied by ystart and ystop, 1882).  It might be less confusing to the user to have "extract_width": 181